CRM/Contact/Form/Edit/Notes.php

```diff
@@ -42,8 +42,8 @@ class CRM_Contact_Form_Edit_Notes {
    */
   public static function buildQuickForm(&$form) {
     $form->applyFilter('__ALL__', 'trim');
-    $form->add('text', 'subject', ts('Subject'), array('size' => 60, 'maxlength' => 254));
-    $form->add('textarea', 'note', ts('Notes'), array('cols' => '60', 'rows' => '3'));
+    $form->addField('subject', array('entity' => 'note', 'size' => '60'));
+    $form->addField('note', array('entity' => 'note', 'rows' => 3));
```

Comment:
No worry, those nitpciks are always valuable reviews. 
For the rows is the default value (which is inside the schema) 4 cols. This is for most textarea's the case, but here it is (and was) 3 cols, so it's overriden there. 
The size of the subject is defined at 60 in the schema, however the getfields api-call returns 45.
This seems because of the schema conversion to the DAO class. The size is defined as 60 in the schema, but gets converted to ```'size' => CRM_Utils_Type::HUGE```, in the generated class, and that constants value is 45. 
The constant class has a todo ```// @todo What's the point of these constants? Backwards compatibility?``` and i think this whole class needs some form of refactoring, so i would leave that here as is, and do that for a follow-up